build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.vanniktech:gradle-maven-publish-plugin:0.8.0'
-        classpath 'com.android.tools.build:gradle:3.4.2'
+        classpath 'com.android.tools.build:gradle:3.5.1'
```

Comment:
I think the update of gradle version and the targetSdkVersion/compileSdkVersion could troubles in such a big library

Reply:
I don't think that should cause an issue. I checked the circleci config file and it hasn't been updated to match the SDK version in the build.gradle file. If changing the circle ci image to 
`docker:`
    `- image: circleci/android:api-29` 
doesn't help, you can add a step to accept the permissions. Example:
`
command: |`
      `yes | sdkmanager --licenses || true
      sdkmanager "platform-tools" "platforms;android-29"
`

Reply:
Thanks for the review. docker image indeed was the problem. Upgraded to api-29